core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -386,8 +389,10 @@ class ReplicaManager(val config: KafkaConfig,
               stateChangeLogger.error(s"Ignoring stop replica (delete=${stopReplicaRequest.deletePartitions}) for " +
                 s"partition $topicPartition due to storage exception", e)
               responseMap.put(topicPartition, Errors.KAFKA_STORAGE_ERROR)
+              stopReplicaCount -= 1
```

Comment:
Why not increment instead of decrement?

Reply:
Oh, because I already had the variable present due to the log where we print out how many partitions we're about to stop:
```
        stateChangeLogger.info(s"Handling stop replica (delete=${stopReplicaRequest.deletePartitions}) for ${partitions.size} partitions (${partitions.mkString(", ")})")
```

Reply:
Is it weird to decrement?

Reply:
Yeah, it's a bit less conventional.